### DISCUSSION TITLE
`dawn kb download` command

### DISCUSSION THREAD

**Discussion by DannyBen** [thesp0nge/dawnscanner#261] [Ideas]
This step from the readme is awkward:

> After that, you need to download the [knowledge base](https://github.com/thesp0nge/dawn_knowledge_base/releases) from Github and unpack the archive to $HOME/dawnscanner/kb directory.

Is there a reason not to have a `dawn kb download` command?

**Comment by thesp0nge** [comment]
nice question... at the moment no, feel free to open me an issue or contributing with a pull request :)
